### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by EpicLPer** [sirWest/AudioSwitch#140]
Seems like the dev didn't update or respond to open issues for almost 4 months now. Did anything happen?

**Comment by adamskovron** [comment]
Windows 10 added simplified native audio switching.  Could be related.

**Comment by EpicLPer** [comment]
But it has no hotkeys, doesn't switch standard communication device and you can't switch your audio recording device either with that.
That's why Audio Switcher would still be nice if it's in active development still :(

**Comment by adamskovron** [comment]
I agree!  Audio Switcher has more features - hopefully the quiet is just momentary

**Comment by GlutenBoy** [comment]
So EpicLPer you would like to be able to assign a hotkey for each device? 

Examples: 

CTRL+ALT+F1 for Speakers
CTRL+ALT+F2 for Headphones

etc ?

**Comment by GlutenBoy** [comment]
I say that because it's already possible to assign hot keys for NextPlaybackDevice, PreviousPlaybackDevice, etc.

**Comment by JFish222** [comment]
I hope it's not dead!

The always visible OSD Selector Menu is great.  I have this on a tv + pc for rapid change between default (mic + TV audio) to Headset (speaker + Mic).  Frankly, this covers a lot of interesting usages elegantly that the windows one does not.

With the adoption of Cortana and Alexa on windows, convenient and complete audio profile switching will become more important!

**Comment by All-github-usernames-taken** [comment]
Well it doesn't surprise me considering his attitude:

https://github.com/sirWest/AudioSwitch/issues/80

**Comment by EpicLPer** [comment]
I rather think it was just a funny answer, hm.
